doc/source/cluster/kubernetes/getting-started/raycluster-quick-start.md

```diff
@@ -48,7 +48,7 @@ kubectl get pods
 kubectl create -k "github.com/ray-project/kuberay/ray-operator/config/default?ref=v1.2.2"
```

Comment:
this still uses v1.2.2, so the operator is still in the `ray-system` namespace.

Reply:
Fixed it. Changed it to master and so does `ray-cluster.sample.yaml`. If there is any consideration, kindly let me know. Additionally, attached the manually testing for your reference.
<img width="1213" alt="test 01" src="https://github.com/user-attachments/assets/8ccd9654-e538-4406-bfe6-35b2007e1c4c" />
<img width="1250" alt="test 02" src="https://github.com/user-attachments/assets/6c40d86f-8edd-4cbb-9b87-bf10b490114d" />